nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/PutHiveQL.java

```diff
@@ -233,6 +234,7 @@ private FunctionContext(boolean rollbackOnFailure, Charset charset, String state
                     }
 
                     // Execute the statement
+                    stmt.setQueryTimeout(context.getProperty(QUERY_TIMEOUT).evaluateAttributeExpressions(flowFile).asInteger());
```

Comment:
If the driver doesn't support this (at least for Apache Hive 1.2.1), it will throw an exception, perhaps wrap these calls in a try/catch and ignore any errors (as they would have been presented to the user via doc and validation)?

Reply:
Good catch, just pushed a commit for PutHiveQL and SelectHiveQL. Thanks!